pkg/controller/secrets/secrets.go

```diff
@@ -65,22 +65,22 @@ func GenerateUserData(privateKey []byte) (*core.Secret, error) {
 			$containerLogsPort = "10250"
 			New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
 			Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-			Install-Module -Force OpenSSHUtils
-			Set-Service -Name ssh-agent -StartupType ‘Automatic’
 			Set-Service -Name sshd -StartupType ‘Automatic’
-			Start-Service ssh-agent
 			Start-Service sshd
 			$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
 			$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
- 			$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
+ 			$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication no'
```

Comment:
Are we sure we want to disable password authentication in this PR? That sort of config change should be a separate PR.

Reply:
why do we want to set the password authentication to no? wouldnt just leaving that line commented as it is by default work?

Reply:
I agree with @aravindhp that we should handle this change in different PR. 
also just setting `PasswordAuthentication no` wont work and we need to set `ChallengeResponseAuthentication no` as well to completely disable password auth.

Reply:
i'm not sure if just keeping it commented out will disable password auth